docs/reference/esql/functions/kibana/docs/base64_decode_tostring.md

```diff
@@ -0,0 +1,11 @@
+<!--
+This is generated by ESQL's AbstractFunctionTestCase. Do no edit it. See ../README.md for how to regenerate it.
+-->
+
+### BASE64_DECODE_TOSTRING
+Decode a base64 string.
+
+```
+row a = "aGVsbG8=" 
+| eval d = base64_decode_tostring(a)
```

Comment:
_tostring prefix is unnecessary - base64_decode/_encode should be enough.
Alternatively use the naming convention used so far is - to/from_base64 . 

The exception is dealing with `binary` fields, which we don't support yet, and are base64 encoded. 
However for those we know the encoding, `to_string(binary)/to_binary(string)` would be the equivalent of the above plus the dedicated datatype.